packages/beacon-node/test/e2e/network/onWorker/dataSerialization.test.ts

```diff
@@ -79,6 +86,25 @@ describe("data serialization through worker boundary", () => {
       request: {method: ReqRespMethod.Status, body: statusZero},
       peer: getValidPeerId(),
     },
+    // New BlockInput events
+    [NetworkEvent.unknownBlockRoot]: {
+      rootHex: toHex(bytes),
```

Comment:
for root should prefer `toRootHex`

```suggestion
      rootHex: toRootHex(bytes),
```

Reply:
updated